semeio/jobs/rft/trajectory.py

```diff
@@ -275,9 +275,7 @@ def load_from_file(cls, filepath):
         filename = os.path.join(filepath)
         if not os.path.isfile(filename):
             raise argparse.ArgumentTypeError(
```

Comment:
I think the raising of an argparse error here was a mistake to be honest. Should we just swap out with an IO error or something? And rather catch that in the entry point (which at that point will throw the correct argparse error).

Reply:
Just to clarify, you would like to keep the entry point as is (except for catching a different error), and change the raised error?

Reply:
_Maybe_ we should actually change the whole configuration and make it more explicit, currently the well file is deduced from a separate file, which is a bit strange.

Reply:
I would very much like to change it, we merely used what was already in place to be honest. But then we'll break all setups already in use.

Reply:
What you did is what I suggested yes

I suppose the ´parse_trajectory_line` would then also need to throw a different error? And that error would need to be caught as well

Reply:
Do you have a preference on the error? I used `ValueError`, but can change it, or make a custom if wanted.

Reply:
`ValueError` should be okay